Gemfile

```diff
@@ -22,6 +22,7 @@ group :production do
 end
 
 group :development do
+  gem 'heroku_rails_deploy', require: false
```

Comment:
Should we add `require: false` to the alerts service Gemfile and the `heroku_rails_deploy` README.md?